docs/uninstall-olm.md

```diff
@@ -0,0 +1,46 @@
+# How do I uninstall OLM?
+
+OLM is installed within a namespace called `openshift-operator-lifecycle-manager` on an OpenShift cluster. On a local minikube cluster, it is installed within the namespace `olm`. You can uninstall OLM in a symmetrical way as you did to install it. 
+
+## Uninstall Released OLM
+For uninstalling release versions of OLM, for example version 0.13.0, you can use the following command:
+
+```bash
+export olm_release=0.13.0
+kubectl delete -f https://github.com/operator-framework/operator-lifecycle-manager/releases/download/${olm_release}/crds.yaml
```

Comment:
I think you'll also have to delete the resources in olm.yaml too to get rid of the cluster scoped resources, which wouldn't get included in deleting the namespace below.

Reply:
Yes, I am aware of that. The tricky thing is that ordering the deletion of those resources is important and we can not use the same ordering as creation. I am wondering if we can just say let's delete the OLM ns and let kubernetes do its job.

Reply:
If I understand you to mean to ensure that you delete everything in the namespace before deleting the namespace, I'm not sure it's worth solving. Can't you just do something like this: https://github.com/operator-framework/operator-lifecycle-manager/blob/ace980894e816e40f26c9831c8eff03935df4f18/scripts/run_e2e_local.sh#L27

Reply:
I went back to deleting `olm` after `crds` method now.